src/pointer/ptr.rs

```diff
@@ -709,14 +730,11 @@ mod _casts {
         ///   at ranges identical to those at which `UnsafeCell`s exist in `*p`
         #[doc(hidden)]
         #[inline]
-        pub unsafe fn cast_unsized_unchecked<U: 'a + ?Sized, F: FnOnce(*mut T) -> *mut U>(
+        pub unsafe fn cast_unsized_unchecked<W: 'a + ?Sized, F: FnOnce(*mut V::Inner) -> *mut W>(
```

Comment:
Here and elsewhere: When `W` is an inner type, not a validity, call it `T` so it doesn't get confused with a validity.

Reply:
[Done](https://github.com/google/zerocopy/compare/07239f8524aef427655ada542b0be8a5f5f54ec3..7655869)